superset/db_engine_specs.py

```diff
@@ -74,6 +84,115 @@ def extra_table_metadata(cls, database, table_name, schema_name):
         """Returns engine-specific table metadata"""
         return {}
 
+    @staticmethod
+    def csv_to_df(names, filepath_or_buffer, sep, header, index_col, squeeze,
```

Comment:
Some linters will trigger on "too many params" on methods. This has lots of boilerplate code here. We can chat about how to improve this using a data structure and an unpack operator `**`, but it may be ok as is. For instance you could have a `pd_read_csv_kwargs` dict that you'd pass around, and ultimately call `pandas.read_csv(**pd_read_csv_kwargs)`